src/server/worldserver/worldserver.conf.dist

```diff
@@ -4424,3 +4425,18 @@ Pvp.FactionBalance.Pct20 = 0.8
 
 #
 ###################################################################################################
+
+###################################################################################################
+# LOAD SETTINGS
+#
+# These settings control content loading
+#
```

Comment:
remove this white line.

Reply:
no, keeping it for logical separation from section description. Same way has been used for PACKET SPOOF PROTECTION SETTINGS and METRIC SETTINGS a bit further up